test/e2e/tests/network/switch-network.spec.ts

```diff
@@ -26,40 +28,27 @@ describe('Switch network - ', function (this: Suite) {
       }) => {
         await loginWithBalanceValidation(driver, ganacheServer);
         const homePage = new HomePage(driver);
-        const headerNavbar = new HeaderNavbar(driver);
-        const selectNetwork = new SelectNetwork(driver);
-        const networkSwitchConfirmation = new ModalConfirmation(driver);
 
         // Validate the default network is Localhost 8545
-        await headerNavbar.check_currentSelectedNetwork('Localhost 8545');
+        await new HeaderNavbar(driver).check_currentSelectedNetwork(
```

Comment:
Could you explain the reason for changing this line? It's currently used only once, but if someone needs it for reference and there are more calls, it could be easily missed. This brings up the question of whether clean code practices are needed. I will leave it to you to decide.

Reply:
If someone needs it for reference, we can recreate the variable easily and quickly. Since we only need it once currently, we don't need to store the variable and use two lines of code for the same objective. The philosophy is don't waste, only keep useful code.